### ISSUE TITLE
Integration test fails after maven downloads: invalid option: --no-rdoc

### ISSUE THREAD

**Issue by olleolleolle** [jruby/warbler#522]
This is what the GH Actions report currently, after a longish Maven interaction:

```
Downloaded: https://repo.maven.apache.org/maven2/org/jruby/jruby-complete/9.2.18.0/jruby-complete-9.2.18.0.jar (26805 KB at 43514.0 KB/sec)
[INFO] ERROR:  While executing gem ... (OptionParser::InvalidOption)
[INFO]     invalid option: --no-rdoc
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:15 min
[INFO] Finished at: 2022-07-06T11:54:33+00:00
[INFO] Final Memory: 62M/189M
[INFO] ------------------------------------------------------------------------
Error:  Failed to execute goal de.saumya.mojo:gem-maven-plugin:1.1.5:initialize (default-initialize) on project warbler: Execution default-initialize of goal de.saumya.mojo:gem-maven-plugin:1.1.5:initialize failed: Java returned: 1 -> [Help 1]
Error: 
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error: 
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
```

**Comment by olleolleolle** [comment]
I was unable to locate a --no-rdoc mentioned with a quick GitHub Search, but I guess there are more ways to locate it.

**Comment by JesseChavez** [comment]
Hi @olleolleolle 

Yes, building warbler fails, I was able to replicate the issue with `rake build`

I've tried to build with JRuby `9.2.21.0` and `9.3.8.0` but the same issue.

After some googling  I discovered the last version the builds `warbler` successully is JRuby `9.2.9.0`, and indeed the build succeeds.

Here is the pull request that fix the issue https://github.com/jruby/warbler/pull/525

**Comment by JesseChavez** [comment]
No sure if the  compilation line should be bumped since only Jave 8+ is supported

```
plugin :compiler, '3.1', :source => '1.6', :target => '1.6'
```

**Comment by JesseChavez** [comment]
After the change, `rake build` succeeds with both JRuby `9.2.21.0` and `9.3.8.0`

```
[INFO] skip non existing resourceDirectory /home/jessec/ibsa/other_gems/warbler/src/test/resources
[INFO] Copying 150 resources
[INFO] Copying 1942 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.1:testCompile (default-testCompile) @ warbler ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-surefire-plugin:3.0.0-M7:test (default-test) @ warbler ---
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default) @ warbler ---
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 56.093 s
[INFO] Finished at: 2022-09-21T07:07:36+10:00
[INFO] Final Memory: 45M/214M
[INFO] ------------------------------------------------------------------------
warbler 2.0.5 built to pkg/warbler-2.0.5.gem.
```

**Comment by olleolleolle** [comment]
Thanks for making this fix! Fixed in #525
